zetta_utils/mazepa/autoexecute_task_queue.py

```diff
@@ -22,36 +26,58 @@ class AutoexecuteTaskQueue(MessageQueue):
     tasks_todo: list[Task] = attrs.field(init=False, factory=list)
     debug: bool = False
     handle_exceptions: bool = False
+    parallel_if_pool_exists: bool = False
 
     def push(self, payloads: Iterable[Task]):
         # TODO: Fix progress bar issue with multiple live displays in rich
         # for task in track(tasks, description="Local task execution..."):
         self.tasks_todo += list(payloads)
 
     def pull(
-        self, max_num: int = 1, max_time_sec: float = 0  # pylint: disable=unused-argument
+        self,
+        max_num: int = 128,
+        max_time_sec: float = 0,  # pylint: disable=unused-argument
```

Comment:
I guess the arg is actually used, so pragma is unnecessary here. Sorry, i think it's my bad for leaving it in